service/src/main/java/com/ge/predix/acs/policy/evaluation/cache/AbstractPolicyEvaluationCache.java

```diff
@@ -311,25 +310,25 @@ private String timestampValue() {
         }
     }
 
-    abstract void delete(final String key);
+    abstract void delete(String key);
```

Comment:
@anurag-gujral "final" is redundant when used for parameters in abstract method declarations (since the concrete method should add "final")